module/documents/item.mjs

```diff
@@ -1020,6 +1022,12 @@ export default class Item5e extends SystemDocumentMixin(Item) {
     // Prepare card data & display it if options.createMessage is true
     const cardData = await item.displayCard(options);
 
+    // Initiate concentration.
+    let effects;
+    if ( config.beginConcentrating ) {
+      effects = await item.actor.beginConcentratingOn(item, { deleteId: config.endConcentration });
+    }
```

Comment:
Perhaps we should split the functionality for ending concentration out of the `beginConcentratingOn` method. I'm thinking ahead to when creatures fail a concentration saving throw, we probably want an easy way to simply end their concentration. Something like `Actor5e#endConcentrationOn(effect)` that takes an effect ID or instance.

```suggestion
    const effects = [];
    if ( config.beginConcentrating ) {
      effects.push(await item.actor.beginConcentratingOn(item));
      if ( config.endConcentration ) effects.push(await item.actor.endConcentrationOn(config.endConcentration));
    }
```

It allows the return value for `beginConcentratingOn` to be a bit nicer also, as it will always simply return the concentration effect that was created, rather than sometimes returning two effects.